pkg/universe.dagger.io/x/vgjm456@qq.com/helm/helm.cue

```diff
@@ -0,0 +1,42 @@
+package helm
+
+import (
+	"dagger.io/dagger"
+	"universe.dagger.io/docker"
+)
+
+#Install: {
+	releaseName: string
+	chartName:   string
+	repoName:    string
+	repository:  string
+	kubeconfig:  dagger.#Secret
+	version:     string | *"latest"
+	commands: [
+		{
+			name: "repo"
+			args: ["add", "\(repoName)", "\(repository)"]
+		},
+		{
+			name: "install"
+			args: ["\(releaseName)", "\(repoName)/\(chartName)"]
```

Comment:
Since `releaseName` is a string, you do not need interpolation, directly insert those in the array.